src/profile-logic/process-profile.js

```diff
@@ -897,8 +904,17 @@ export function serializeProfile(
   includeNetworkUrls: boolean = true
 ): string {
   // stringTable -> stringArray
+  let urlCounter = 0;
   const newProfile = Object.assign({}, profile, {
     meta: { ...profile.meta, networkURLsRemoved: !includeNetworkUrls },
+    pages:
+      includeNetworkUrls === false
```

Comment:
Oh, I just noticed this after I approved. Can you add a test for this behavior? You could follow-up with it, but I think it's important to make sure we are not regressing on PII sanitation.

Reply:
Okay, I will write some test for that in a follow-up.